web/sites/default/config/core.extension.yml

```diff
@@ -18,6 +18,7 @@ module:
   chosen_lib: 0
   ckeditor: 0
   ckeditor_config: 0
+  clamav: 0
```

Comment:
I think this might have had a config export done before an initial `lando refresh` that includes a config import. 🤔